apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-global-styles-tab-selected.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+const getTabNameFromId = ( id ) => ( id?.endsWith?.( '-block' ) ? 'block-type' : 'root' );
+
+export function trackGlobalStylesTabSelected( { tab, open } = {} ) {
+	const activeTabElement = document.querySelector(
+		'.edit-site-global-styles-sidebar .components-tab-panel__tabs button.is-active'
+	);
+	const activeTabName = tab || getTabNameFromId( activeTabElement?.id );
+	const isSidebarOpen =
+		typeof open !== 'undefined'
+			? open
+			: !! document.querySelector( '.edit-site-global-styles-sidebar' );
+
+	tracksRecordEvent( 'wpcom_block_editor_global_styles_tab_selected', {
+		tab: activeTabName,
+		open: isSidebarOpen,
+	} );
+}
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_global_styles_tab_selected`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export default () => ( {
+	selector: `.edit-site-global-styles-sidebar .components-tab-panel__tabs button:not(.is-active)`,
+	type: 'click',
+	capture: true,
```

Comment:
Was there a reason this needed to happen on capture?

Reply:
Yes! We need it because of the `:not(.is-active)` part of the selector. If this runs on bubble rather than capture, then it will already be `.is-active` before this event listener runs. Basically running this on `bubble` makes it impossible to only run the event listener when the tab is not active at the time of the click. I'll add a comment. 

You can test this by removing `capture` and running E2E tests. `should not trigger the event when clicking on the already active tab` test will fail.

Reply:
Makes sense! 😁